experimental/PropertyDDS/packages/property-binder/src/data_binder/data_binder.js

```diff
@@ -137,17 +137,14 @@ const _popUserData = function(in_context) {
 };
 
 /**
- * A DataBinder allows one to register a number of bindings for different HFDM property types. The
+ * A DataBinder allows one to register a number of bindings for different property types. The
  * DataBinder can then be bound to
  * a {@link https://pages.git.autodesk.com/LYNX/HFDM_SDK/doc/latest/LYNX.Property.Workspace.html|Workspace} to have the
```

Comment:
There is still an HFDM reference here, but I think I already fixed this section in my other PR.